apps/condo/index.js

```diff
@@ -143,6 +143,53 @@ const apps = () => {
 
 /** @type {(app: import('express').Application) => void} */
 const extendExpressApp = (app) => {
+    const runtimeStats = {
+        activeRequestsIds: new Set(),
+        activeRequestsCountByType: {
+            graphql: 0,
+            oidc: 0,
+            other: 0,
+        },
+    }
+
+    app.use(function runtimeStatsMiddleware (req, res, next) {
+        const url = req.url
+        const method = (req.method || 'get').toLowerCase()
+
+        const isPost = method === 'post'
+
+        const isAdminApi = url.startsWith('/admin/api')
+        const isOidc = url.startsWith('/oidc')
+
+        /** @type {'graphql' | 'oidc' | 'other'} */
+        let requestType = 'other'
+
+        if (isPost && isAdminApi) requestType = 'graphql'
+        if (isOidc) requestType = 'oidc'
```

Comment:
Add “well-known” group and “health-check” group

Reply:
And also: https://github.com/open-condo-software/condo/blob/main/apps/condo/index.js#L134

/api/version /api/files /api/auth … may be something others

It’s ok to extract “files” and “others-api” group by /api prefix.

Reply:
A new group `api` was added

Reply:
`wellKnown` and `healthCheck` groups too